### PR TITLE
Yes, You Can Do AI with C++ blogpost

### PR DESCRIPTION
**Pull request by czoido**

### CODE REVIEW

File: _posts/2024-12-20-You-can-do-AI-with-cpp.markdown [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+Then, use a [Conan deployer](https://docs.conan.io/2/reference/extensions/deployers.html)
+to move the installed files from the Conan cache to the user space. To accomplish this,
+simply run the following command:
+
+```shell
+# Install llama-cpp using Conan and deploy to the local folder
+$ conan install --requires=llama-cpp/b4079 --build=missing \
+                -o="llama-cpp/*:with_examples=True" \
+                -o="llama-cpp/*:with_curl=True" \
+                --deployer=full_deploy
```
Comment (czoido) [inline _posts/2024-12-20-You-can-do-AI-with-cpp.markdown:60]:
```suggestion
                --deployer=direct_deply
```

File: _posts/2024-12-20-You-can-do-AI-with-cpp.markdown [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+post](https://blog.conan.io/2023/05/11/tensorflow-lite-cpp-mobile-ml-guide.html)
+showcasing how to build a real-time human pose detection application using TensorFlow Lite
+and OpenCV. Check it out if you haven't read it yet.
+
+One of the interesting aspects of using the library is the availability of numerous models
+on platforms like [Kaggle Models](https://www.kaggle.com/models) for various tasks, which
+can be easily integrated into your code. For more information on Tensorflow Lite, please
+[check their documentation](https://www.tensorflow.org/lite/guide).
+
+#### ONNX Runtime
```
Comment (memsharded) [inline _posts/2024-12-20-You-can-do-AI-with-cpp.markdown:131]: Maybe add links to the conan packages, or at least a ``conan install --requires=`` or something like that?

File: _posts/2024-12-20-You-can-do-AI-with-cpp.markdown [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+   to get started quickly.
+
+Additionally, ONNX Runtime offers multiple options for tuning performance using various
+runtime configurations or hardware accelerators. Explore [the Performance section in the
+documentation](https://onnxruntime.ai/docs/performance/) for more details. For more
+information, visit the [ONNX Runtime documentation](https://onnxruntime.ai/docs/).
+
+#### OpenVINO
+
+**OpenVINO** (Open Visual Inference and Neural Network Optimization) is an
```
Comment (memsharded) [inline _posts/2024-12-20-You-can-do-AI-with-cpp.markdown:155]: Same, links to Conan packages?

**Review by memsharded — APPROVED** [review]
I love it.

Maybe just add a couple of links, or maybe just in the conclusion add: "search for these libraries in conan.io/center or using ``conan list <pattern>`` with the conan client"

**Comment by czoido** [comment]
> I love it.
> 
> Maybe just add a couple of links, or maybe just in the conclusion add: "search for these libraries in conan.io/center or using `conan list <pattern>` with the conan client"

Done!

File: _posts/2024-12-23-You-can-do-AI-with-cpp.markdown [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+As you can see, in just a few minutes, we can have our own LLM running locally, all using
+C++. You can also use the libraries provided by the **llama-cpp** Conan package to
+integrate LLMs into your own applications. For example, here is the code for the
+[llama-cli](https://github.com/ggerganov/llama.cpp/blob/b4079/examples/main/main.cpp) that
+we just executed. For more information on the LLaMA.cpp project, please [check their
+repository on GitHub](https://github.com/ggerganov/llama.cpp).
+
+#### [TensorFlow Lite](https://conan.io/center/recipes/tensorflow-lite)
+
+**TensorFlow Lite** is a specialized version of [TensorFlow](https://www.tensorflow.org/)
```
Comment (AbrilRBS) [inline _posts/2024-12-23-You-can-do-AI-with-cpp.markdown:105]: Maybe something like `(now LiteRT)`?

**Review by AbrilRBS — APPROVED** [review]
Just to clarify, here we don't get bitten by unicode issues with the ' and ’ as in the docs, right?

File: _posts/2024-12-23-You-can-do-AI-with-cpp.markdown (hunk 4 above)
Comment (czoido) [reply _posts/2024-12-23-You-can-do-AI-with-cpp.markdown:105]: The thing is that we don't have LiteRT in Conan Center yet... 🤔

**Comment by czoido** [comment]
> Just to clarify, here we don't get bitten by unicode issues with the ' and `’` as in the docs, right?

It renders fine for me, I think it's ok

**Review by danimtb — APPROVED** [review]

**Pull request merged**